ambry-api/src/main/java/com.github.ambry/account/Container.java

```diff
@@ -56,44 +56,50 @@
   public static final short UNKNOWN_CONTAINER_ID = -1;
 
   /**
-   * The id of {@link #UNKNOWN_PUBLIC_CONTAINER}.
+   * The id for the containers to be associated with the blobs that are put without specifying a target container,
+   * but are specified public. {@link #DEFAULT_PUBLIC_CONTAINER} is one of the containers that use it.
    */
-  public static final short UNKNOWN_PUBLIC_CONTAINER_ID = -2;
+  public static final short DEFAULT_PUBLIC_CONTAINER_ID = 0;
 
   /**
-   * The id of {@link #UNKNOWN_PRIVATE_CONTAINER}.
+   * The id for the containers to be associated with the blobs that are put without specifying a target container,
+   * but are specified private. {@link #DEFAULT_PRIVATE_CONTAINER} is one of the containers that use it.
    */
-  public static final short UNKNOWN_PRIVATE_CONTAINER_ID = -3;
+  public static final short DEFAULT_PRIVATE_CONTAINER_ID = 1;
 
   /**
    * The name of {@link #UNKNOWN_CONTAINER}.
    */
-  public static final String UNKNOWN_CONTAINER_NAME = "ambry-unknown-container-name";
+  public static final String UNKNOWN_CONTAINER_NAME = "ambry-unknown-container";
```

Comment:
why do we need prefix of "ambry" in the values? Its understood that "Accounts" and "Containers" makes sense only for Ambry.